ocpp/v201/call.py

```diff
@@ -0,0 +1,429 @@
+from typing import Dict, List
+from dataclasses import dataclass
+
+
+@dataclass
+class AuthorizeRequestPayload:
+    id_token: Dict
+    certificate: str = None
+    iso15118_certificate_hash_data: List = None
```

Comment:
Two comments that are meant for all classes in this file:

* Can you remove the term  'Request' from the class name.
* Can you fix the type hints for all attributes that could be `None` and use `Optional`? So `certificate: Optional[str] = None`.